diff -purN khtml-5.48.0.org/CMakeLists.txt khtml-5.48.0/CMakeLists.txt
--- khtml-5.48.0.org/CMakeLists.txt	2018-07-07 22:30:52.000000000 +0000
+++ khtml-5.48.0/CMakeLists.txt	2018-07-26 04:25:38.950376906 +0000
@@ -75,13 +75,6 @@ set_package_properties(JPEG PROPERTIES D
                        PURPOSE "Required for decoding and displaying JPEG images"
                       )
 
-find_package(GIF)
-set_package_properties(GIF PROPERTIES DESCRIPTION "GIF decoding library"
-                       URL "http://sourceforge.net/projects/giflib"
-                       TYPE REQUIRED
-                       PURPOSE "Required for decoding and displaying GIF images"
-                      )
-
 find_package(PNG)
 set_package_properties(PNG PROPERTIES DESCRIPTION "PNG decoding library"
                        URL "http://www.libpng.org/pub/png"
diff -purN khtml-5.48.0.org/src/CMakeLists.txt khtml-5.48.0/src/CMakeLists.txt
--- khtml-5.48.0.org/src/CMakeLists.txt	2018-07-07 22:30:52.000000000 +0000
+++ khtml-5.48.0/src/CMakeLists.txt	2018-07-26 04:26:03.523376564 +0000
@@ -439,7 +439,6 @@ set(khtmlimload_STAT_SRCS
 set(decoders_STAT_SRCS
   imload/decoders/jpegloader.cpp
   imload/decoders/pngloader.cpp
-  imload/decoders/gifloader.cpp
   imload/decoders/qimageioloader.cpp
 )
 
--- khtml-5.48.0/src/imload/imagemanager.cpp~	2018-07-07 22:30:52.000000000 +0000
+++ khtml-5.48.0/src/imload/imagemanager.cpp	2018-07-26 04:37:25.449367070 +0000
@@ -58,7 +58,6 @@
 {
     loaderDB->registerLoaderProvider(new JPEGLoaderProvider);
     loaderDB->registerLoaderProvider(new PNGLoaderProvider);
-    loaderDB->registerLoaderProvider(new GIFLoaderProvider);
     loaderDB->registerLoaderProvider(new QImageIOLoaderProvider);
 }
 
